main.tex

```diff
@@ -161,6 +161,32 @@
 
 % -------------------------- USER SETTINGS ---------------------------
 
+\usepackage{float}
+\usepackage{listings}
+\usepackage{xcolor}
+\usepackage{menukeys}
+\usepackage{array}
+
+\lstdefinelanguage{json}{
```

Comment:
do we need to specify this manuall? like \begin{lstlisting}[language=Python, caption=Python example] in https://www.overleaf.com/learn/latex/Code_listing

Reply:
I don't think there is a preset for json. Also the one for python is ugly imo.